src/box/lua/config/source/env.lua

```diff
@@ -35,11 +43,20 @@ function methods.get(self)
     return self._values
 end
 
-local function new()
+local function new(opts)
+    local opts = opts or {}
+    local env_var_suffix = opts.env_var_suffix
+
+    if env_var_suffix ~= nil then
+        env_var_suffix = '_' .. env_var_suffix:upper()
+    end
+
     return setmetatable({
         name = 'env',
```

Comment:
I would like to make the names of the standard configuration sources unique. Let's add the suffix into the name.

Reply:
Fixed this way:

```diff
diff --git a/src/box/lua/config/source/env.lua b/src/box/lua/config/source/env.lua
index 0adbf52732..735fb17aa2 100644
--- a/src/box/lua/config/source/env.lua
+++ b/src/box/lua/config/source/env.lua
@@ -47,12 +47,15 @@ local function new(opts)
     local opts = opts or {}
     local env_var_suffix = opts.env_var_suffix
 
+    local name = 'env'
+
     if env_var_suffix ~= nil then
+        name = ('%s (%s)'):format(name, env_var_suffix)
         env_var_suffix = '_' .. env_var_suffix:upper()
     end
 
     return setmetatable({
-        name = 'env',
+        name = name,
         type = 'instance',
 
         _values = {},
```